templates/graphdb-master.yaml

```diff
@@ -59,29 +74,41 @@ spec:
           volumeMounts:
```

Comment:
Consider adding extra volume mounts, extra envs & extra envFrom in order to easily extend & override chart configurations.

Reply:
will create a task